hooks/useCart.js

```diff
@@ -251,6 +251,31 @@ export default function useCart(initialCart = false, options = {}) {
     }
   };
 
+  const applyIsic = async (surname, code, callback) => {
+    setIsLoading(true);
+    try {
+      const response = await fetch(`/api/lib/v1/cart/isic`, {
+        method: "POST",
+        body: JSON.stringify({
+          isic: {
+            surname,
+            code,
+          },
+        }),
+      }).then((result) => {
+        if (isFunction(callback)) {
+          callback(result);
+        }
+        return result.json();
+      });
```

Comment:
@rasevp naco som ti pisal, ako mame pracovat s callbackom, ked na druhy den to naprogramujes po starom?!

Reply:
upravene